openpype/hosts/maya/plugins/publish/extract_thumbnail.py

```diff
@@ -92,7 +92,8 @@ def process(self, instance):
             "Create temp directory {} for thumbnail".format(dst_staging)
         )
         # Store new staging to cleanup paths
-        instance.context.data["cleanupFullPaths"].append(dst_staging)
+        if not instance.data.get("stagingDir_persistent", False):
```

Comment:
Why this? It is in temp directory, it will keep a lot of files for windows users...

Reply:
This is part of the feature design; https://openpype.io/docs/project_settings/settings_project_global#custom-staging-directory-profiles

If a user enables persistent staging, it will be their responsibility to clean up any files.

Reply:
But this is not in custom staging dir. This is in tempfolder...

Reply:
Yeah, good point. Have removed.